scripts/speech_recognition/canary/build_canary_1b_special_tokenizer.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
```

Comment:
I don't believe this script is necessary. It can just be an example in the canary tutorial instead.

Reply:
The people building the new tokenizer for Canary 3 or community Canary forks will have an easier time and won't need to guess how the previous special tokenizers were built :)

Reply:
Those people will be using the tutorial anyhow no? From my understanding, Canary use hasn't picked up that much this year, so there's not a significant community. It's more likely you'll have greater growth with the current model, so providing backend support may not be crucial.

Reply:
I'm re-doing the Canary 2 tokenizer right now and am grateful I put these scripts in here. Not removing them :)